.github/workflows/test.yml

```diff
@@ -226,14 +226,60 @@ jobs:
           name: generated_binding-${{ matrix.config.os }}-R-${{ matrix.config.r }}-rust-${{ matrix.config.rust-version }}-${{ matrix.config.target || 'default'}}
           path: generated_bindings
 
-      # Run tests again using different bindings
-      - name: Run tests on precomputed bindings shipped with libR-sys
-        if: matrix.config.skip-tests != 'true'
-        run: |
-          . ./ci-cargo.ps1
-          ci-cargo test -vv $(if ($env:RUST_TARGET -ne '') {"--target=$env:RUST_TARGET"} ) '--' --nocapture -ActionName "Running tests for target: $env:RUST_TARGET"
-        env:
-          RUST_TARGET: ${{ matrix.config.target }}
+      # # Run tests again using different bindings
+      # - name: Run tests on precomputed bindings shipped with libR-sys
+      #   if: matrix.config.skip-tests != 'true'
+      #   run: |
+      #     . ./ci-cargo.ps1
+      #     ci-cargo test -vv $(if ($env:RUST_TARGET -ne '') {"--target=$env:RUST_TARGET"} ) '--' --nocapture -ActionName "Running tests for target: $env:RUST_TARGET"
+      #   env:
+      #     RUST_TARGET: ${{ matrix.config.target }}
```

Comment:
Why are these commented out? Is it incase its easier to go backwards and uncomment it? Maybe its best to just delete?

Reply:
I think we need to address this. This is the experience "everyone" else experience. Although this situation should be checked, in the first workflow, this is more like an extra check. But this fails. It fails because it doesn't use the newly generated bindings, because it can't.